### PR TITLE
Add album-cover related gradients

### PR DESCRIPTION
**Pull request by GalexY727**
This is most definitely a personal change, much like #23, but hey! I like colors and others might too. It does add a new dependency of colorthief, which is worth mentioning. Here's a gif preview :)
Gif example

(It pulls colors from the album cover to make a palette for the song title, artist name, and eq bars)

### CODE REVIEW

File: api/spotify.py [hunk 1]
```diff
@@ -111,6 +117,9 @@ def makeSVG(data, background_color, border_color):
     barCount = 84
     contentBar = "".join(["<div class='bar'></div>" for _ in range(barCount)])
     barCSS = barGen(barCount)
+    barPalette = gradientGen(data["item"]["album"]["images"][1]["url"], 4)
```
Comment (novatorem) [inline api/spotify.py]: These two lines would error out when the user isn't actively listening to anything.
Probably best to make use of the two checks below (if playing + if album art exists)

```
[ERROR] Exception on /api/spotify [GET]
Traceback (most recent call last):
  File "/var/task/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/var/task/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/var/task/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/var/task/flask/_compat.py", line 39, in reraise
    raise value
  File "/var/task/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/var/task/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "./api/spotify.py", line 175, in catch_all
    svg = makeSVG(data, background_color, border_color)
  File "./api/spotify.py", line 120, in makeSVG
    barPalette = gradientGen(data["item"]["album"]["images"][1]["url"], 4)
KeyError: 'item'
```

Comment (GalexY727) [reply api/spotify.py]: Fixed, sorry about that one, I noticed it today

Comment (GalexY727) [reply api/spotify.py]: I added a PLACEHOLDER_URL as a band-aid to that, I completely overlooked those two checks :eyes:

**Comment by GalexY727** [comment]
It would be honestly better to use the placeholder image that is in the code there, but I am not 100% sure if colorthief will recognize b64

**Comment by GalexY727** [comment]
Feel free to omit changes since commit [836b7b4](https://github.com/novatorem/novatorem/pull/66/commits/836b7b4f9069eed5b0ba8f72eb840d56e189063c)

Edit: Those commit messages are horrifying... I-- I apologize.
All I did was enable the status message, make that text gradient (of course), then hide EQ bars when not listening to music

**Comment by novatorem** [comment]
> Feel free to omit changes since commit [836b7b4](https://github.com/novatorem/novatorem/pull/66/commits/836b7b4f9069eed5b0ba8f72eb840d56e189063c)
> 
> Edit: Those commit messages are horrifying... I-- I apologize. All I did was enable the status message, make that text gradient (of course), then hide EQ bars when not listening to music

That's cool, it'll be a squash merge. Don't want to play around with cherry-picking a range nor rebasing, so I'll roll out your personalized changes.

Thank you for the work involved!

**Pull request merged**